pkg/upstream/replicated.go

```diff
@@ -585,7 +585,9 @@ func createConfigValues(applicationName string, config *kotsv1beta1.Config, exis
 
 			for _, repeatableValues := range item.ValuesByGroup {
 				for name, value := range repeatableValues {
-					err := createConfigValue(newValues, builder, builderOptions, name, value, value, item.Name)
+					// currently, `ValuesByGroup` defined in the config contains the defaults for repeatable items.
+					// so we pass the value as the item default, and an empty string for the item value here.
+					err := createConfigValue(newValues, builder, builderOptions, name, "", value, item.Name)
```

Comment:
the change makes sense, but what about if we change it to be `for name, default := range` to indicate what you also describe in the comment?